packages/patternfly-4/src/pages/index.js

```diff
@@ -1,20 +1,215 @@
 import React from 'react';
 import Layout from '../components/layout';
+import './styles.scss';
+import './footer.scss';
 import SEO from '../components/seo';
+import { Link } from 'gatsby';
 import {
+  Grid,
+  GridItem,
+  Nav,
+  NavList,
+  NavItem,
   PageSection,
   PageSectionVariants,
-  Bullseye
+  Title,
+  TextContent,
+  Text,
+  TextVariants
 } from '@patternfly/react-core';
-import orb from '../images/patternfly-orb.svg';
+import { CaretRightIcon } from '@patternfly/react-icons';
+import orb from '../images/PF4_logo.svg';
+import patternflyLogo from '../images/l_pf-reverse-164x11.png';
+import pf4hardware from '../images/PF4hardware.svg';
+import principles from '../images/PF4_WIREFRAME.png';
+import redhatLogo from '../images/RHLogo.svg';
 
 const IndexPage = () => (
   <Layout>
     <SEO title="Home" keywords={['gatsby', 'application', 'react']} />
-    <PageSection variant={PageSectionVariants.darker} style={{height: 'calc(100vh - 76px)'}}>
-      <Bullseye>
-        <img src={orb} alt="PatternFly logo" />
-      </Bullseye>
+    <PageSection variant={PageSectionVariants.light} className="pf4-c-background-image pf-u-display-flex pf-u-justify-content-center pf-u-align-items-center">
+      <Grid>
+        <GridItem sm={12} md={6} mdOffset={3} className="pf-u-text-align-center">
+          <TextContent>
+            <img src={orb} alt="PatternFly logo" />
+            <Title size="4xl" className="pf-m-white">Build better experiences with repeatable, scalable design.</Title>
+            <Title size="xl" className="pf-m-white pf-u-mb-md pf-u-mb-3xl-on-md">PatternFly 4 is an open source design system built to drive consistency and unify teams.</Title>
+            <Link to="/get-started/about">
+              <button type="button" aria-label="Get started with PatternFly" className="pf4-c-button__cta-primary pf-u-mr-0 pf-u-mr-sm-on-sm pf-u-mb-md pf-u-mr-xl-on-md">Get Started</button>
```

Comment:
Between a window width of ~760 to ~840 px they were stacked but not centered

Reply:
This should be fixed now - I set it to stack on devices that fall under the `small` media query.